packages/wrangler/src/deployment-bundle/provision.ts

```diff
@@ -0,0 +1,64 @@
+import type { Config } from "../config";
+import type { CfWorkerInit } from "./worker";
+
+/**
+ * A symbol to inherit a binding from the deployed worker.
+ */
+export const INHERIT_SYMBOL = Symbol.for("inherit_binding");
+
+export function getBindings(
+	config: Config | undefined,
```

Comment:
Why is config optional here?

Reply:
It could be undefined for pages project.